### PR TITLE
Change package author

### PR DESCRIPTION
**Pull request by garymm**
Microsoft doesn't author this package, the ONNX community does.

Copied from onnx/onnx:
https://github.com/onnx/onnx/blob/acc127219b45bc27b0180b1fdc08299eac81b167/setup.py#L342

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -70,16 +70,16 @@ def run(self):
...
     description='Tensorflow to ONNX converter',
     setup_requires=['pytest-runner'],
     tests_require=['graphviz', 'parameterized', 'pytest', 'pytest-cov', 'pyyaml'],
     cmdclass=cmdclass,
     packages=find_packages(),
     license='Apache License v2.0',
-    author='onnx@microsoft.com',
-    author_email='onnx@microsoft.com',
+    author='ONNX',
+    author_email='onnx-technical-discuss@lists.lfai.foundation',
```
Comment (fatcat-z) [inline setup.py:82]: In https://lists.lfaidata.foundation/g/onnx-technical-discuss, I saw the email should be something like below:

onnx-technical-discuss@lists.lfaidata.foundation

Could you please confirm the domain is "lfai" or "lfaidata"?

Comment (garymm) [reply setup.py]: Thanks for catching. Done here. I will also update in onnx/onnx.

**Comment by hwangdeyu** [comment]
It should be signed off for each of commit in tf2onnx.

**Review by hwangdeyu — APPROVED** [review]
LGTM. Please fix CI DCO required. Thanks!

**Comment by garymm** [comment]
DCO check passed. Please merge. Thanks!

**Pull request merged**